Bottom: 83401534249699b3aaa0dce2863db1dbe3c25ea2
Top:    5bd81bc351caaeb882c5f12d74ea1de314ee315f
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-19 14:35:03 +0800

Refresh of introduce-skb_ring

---

diff --git a/include/linux/skb_ring.h b/include/linux/skb_ring.h
index caa759b..9666451 100644
--- a/include/linux/skb_ring.h
+++ b/include/linux/skb_ring.h
@@ -47,7 +47,7 @@ int skb_ring_peek(struct skb_ring *ring)
 	unsigned long tail = ACCESS_ONCE(ring->tail);
 	int ret = 0;
 
-	if (CIRC_SPACE(head, tail, ring->size) >= 1)
+	if (CIRC_CNT(head, tail, ring->size) >= 1)
 		ret = ring->descs[tail].len;
 
 	return ret;
